lib/Macros/CMakeLists.txt

```diff
@@ -76,3 +76,4 @@ endfunction()
 
 add_subdirectory(Sources/SwiftMacros)
 add_subdirectory(Sources/ObservationMacros)
+add_subdirectory(Sources/DistributedMacros)
```

Comment:
Oh okey happy to move over there, thanks

Reply:
Actually, I don't think we can -- since the macro declarations use Distributed types... so that'd be weird perhaps...?

Maybe doesn't matter, I'll try once I have them working

Reply:
Done, it'll just be the `@DistributedProtocol` now